File: week_06/day_3/composition/Composition_Lesson_End/app/src/test/java/example/codeclan/com/composition_lesson_end/OgreTest.java
package example.codeclan.com.composition_lesson_end;

/**
 * Created by user on 28/08/2017.
 */

import static org.junit.Assert.*;
import org.junit.*;

import example.codeclan.com.composition_lesson_end.wizard_management.Ogre;


public class OgreTest {

    Ogre ogre;

    @Before
    public void before(){
        ogre = new Ogre("Freddie");
    }

    @Test
    public void hasName(){
        assertEquals("Freddie", ogre.getName());
    }
}

File: week_10/c_sharp_course/week_1/day_4/composition/starter_code/wizard_management_specs/OgreTest.cs
using NUnit.Framework;

namespace WizardManagement { 

  [TestFixture]
  public class OgreTest {
    Ogre ogre;

    [SetUp]
    public void Init() 
    {
      ogre = new Ogre("Freddie");
    }

    [Test]
    public void HasName()
    {
      Assert.AreEqual("Freddie", ogre.Name);
    }
  }
}